src/Http/Remote.php

```diff
@@ -60,6 +60,8 @@ public function __call(string $method, array $arguments = [])
 
 	/**
 	 * Constructor
+	 *
+	 * @throws Exception when the curl request failed
```

Comment:
```suggestion
	 * @throws \Exception when the curl request failed
```

I think all these need to have a leading `\` to declare that the top-level Exception is thrown, not one from the Http namespace (which PHP reflections/docblock parser will assume otherwise)

Reply:
I agree with @distantnative. I've added suggestions for the other parts, so you can simply apply them all.